create_graphics.py

```diff
@@ -32,6 +33,11 @@
 
 AIRPORTS = 'static/Airports_locs.txt'
 
+COMBINED_FN = 'combined_{fhr:03d}.grib2'
+TMP_FN = 'combined_{fhr:03d}.tmp.grib2'
```

Comment:
Using parameters here because multiple methods need to know how these temporary files are named.